apps/smartling/frontend/src/Sidebar.tsx

```diff
@@ -204,6 +207,11 @@ export default class Sidebar extends React.Component<Props, State> {
     const { sdk, projectId } = this.props;
     const { token } = this.state;
 
+    if (!token) {
+      this.runAuthFlow(true);
+      return;
```

Comment:
nit: do we want anything helpful to say/throw?

Reply:
No I think this is OK because it will kick back into the runAuthFlow, grab a token and come back through normally.. if it makes it passed here there is a logic gate on line 217 that fails through the same way

```typescript
if (res.code === 'AUTHENTICATION_ERROR') {
  this.runAuthFlow(true);
}
```